terraform/aws/scenarios/omnibus-tiered-upgrade/main.tf

```diff
@@ -189,14 +229,12 @@ resource "null_resource" "front_end_upgrade" {
     host = module.front_end.public_ipv4_dns
   }
 
-  # upgrade chef-server
+  # reconfigure on frontend chef-server
   provisioner "remote-exec" {
     inline = [
       "set -evx",
-      "echo -e '\nBEGIN UPGRADE CHEF SERVER (FRONT-END)\n'",
-      "curl -vo /tmp/${replace(var.upgrade_version_url, "/^.*\\//", "")} ${var.upgrade_version_url}",
-      "sudo ${replace(var.upgrade_version_url, "rpm", "") != var.upgrade_version_url ? "rpm -U" : "dpkg -iEG"} /tmp/${replace(var.upgrade_version_url, "/^.*\\//", "")}",
-      "sudo CHEF_LICENSE='accept' chef-server-ctl upgrade",
+      "echo -e '\nEND UPGRADE CHEF SERVER (FRONT-END)\n'",
+      "sudo CHEF_LICENSE='accept' chef-server-ctl reconfigure",
```

Comment:
Let's put a comment here explaining that this isn't what the directions say but that we think the current directions may be impacted by a bug.